collectors/metrics/pkg/logger/logger.go

```diff
@@ -34,27 +35,27 @@ func LogLevelFromString(l string) level.Option {
 
 // Log is used to handle the error of logger.Log globally
 func Log(log log.Logger, l LogLevel, keyvals ...interface{}) {
-	errkey := "failover_err_%d"
+	//errkey := "failover_err_%d"
 	switch l {
 	case Debug:
 		err := level.Debug(log).Log(keyvals...)
 		if err != nil {
-			fmt.Sprintf(errkey, err)
+			fmt.Fprintf(os.Stderr, "Error logging: %v\n", err)
```

Comment:
I'm not sure I understand the reasoning for this.
The earlier `fmt.Sprint` is wrong for sure, but even then why do we need to error out for logging errors? We usually don't have to check this go-kit logger and exclude it from errcheck in other upstreams.